### PR TITLE
bubble-sort

### PR DESCRIPTION
**Pull request by AlanSoto31**
This project was based on creating a sort method using bubble sort, where two elements are compared and the bigger one will be moved one spot to the right until it becomes the biggest element. 
we created two methods one takes numbers and will sort them by using the bubble sort method 
and the second one takes a string and sort it as well we used to yield to call part of the code we were given to apply it to our second method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
-# bubble-sort
\ No newline at end of file
+# bubble-sort
```
Comment (SafaErden) [inline README.md]: - [x] Please capitalize your README title and remove the dash sign between the words.

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+
+This project was based on creating a sort method using bubble sort, where two elements are compared and the bigger one will be moved one spot to the right until it becomes the biggest element. 
+
+## Built With
+
+- Ruby
+
+## Prerequisites 
+
+- Chrome browser
```
Comment (SafaErden) [inline README.md]: - [x] If users really need a Chrome browser, please explain the reason it on your README.

File: README.md [hunk 3]
```diff
@@ -1 +1,42 @@
...
+
+## Set up
```
Comment (SafaErden) [inline README.md]: - [ ] Populate the Setup section on your README to give a better description of how to use your app.

File: bubble.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  k = arr.length
+  while k.positive?
+    n = 0
+    while n < arr.length
+      arr[n], arr[n + 1] = arr[n + 1], arr[n] if !arr[n + 1].nil? && arr[n] > arr[n + 1]
+      n += 1
+    end
+    k -= 1
+  end
+  print arr
```
Comment (SafaErden) [inline bubble.rb]: - [x] Instead of printing the result, please return the sorted array.
image

File: bubble.rb [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  k = arr.length
+  while k.positive?
+    n = 0
+    while n < arr.length
+      arr[n], arr[n + 1] = arr[n + 1], arr[n] if !arr[n + 1].nil? && yield(arr[n], arr[n + 1]).positive?
+      n += 1
+    end
+    k -= 1
+  end
+  print arr
```
Comment (SafaErden) [inline bubble.rb]: - [x]  Instead of printing the result, please return the sorted array.
image

**Review by SafaErden — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️

**Hi TEAM, your project is awesome. 👐 However, we need a little fix before we can proceed. Kindly find the comments below ⤵**
##

**REQUIRED CHANGES**

- [x] Your sorting methods are working but you should return the sorted array instead of printing them. Please see the details below:

##

**NOTES**

Friendly reminder: Remember it's important to have a professional repo so you can increase your job opportunities.
Please take a step back and read the lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.

So please use imperative tense for your commit messages in your upcoming projects.
##

**SUMMARY**

As I mentioned above, your project is almost done. You have some minor issues to solve. Please be an optimist and do not forget that the main purpose is to learn and practice. Do not let the requested changes discourage you 👍

##

*Kindly make the changes and submit another code review once you are ready💪*

Questions or comments? Reach me on [Twitter](https://twitter.com/safaerden) or [LinkedIn](https://www.linkedin.com/in/safaerden/) 🗨️

Happy coding_ 👨‍💻 

**Safa ERDEN**

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****

_P.S. Your README isn't really descriptive. Please add instructions for cloning your project and running it._ 

 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**